modules/dreamview/backend/dreamview.cc

```diff
@@ -57,8 +57,11 @@ Status Dreamview::Init() {
   // javascripts and handles websocket requests.
   std::vector<std::string> options = {
       "document_root",      FLAGS_static_file_dir,   "listening_ports",
-      FLAGS_server_ports,   "websocket_timeout_ms",  FLAGS_websocket_timeout_ms,
-      "request_timeout_ms", FLAGS_request_timeout_ms};
+      FLAGS_server_ports,   "websocket_timeout_ms",  "3600",
+      "request_timeout_ms", "3600",
```

Comment:
why change the gflag to literal for the two timeout settings above? 3600ms is also a weird choice